dotCMS/src/main/webapp/html/portlet/ext/contentlet/contentlet_actions_inc.jsp

```diff
@@ -118,9 +118,10 @@ catch(Exception e){
 	
 	<a onclick="contentAdmin.executeWfAction('<%=action.getId()%>', <%= hasPushPublishActionlet || action.isAssignable() || action.isCommentable() || UtilMethods.isSet(action.getCondition()) %>)">
 	<span class="<%=action.getIcon()%>"></span>
-		<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, action.getName())) +"<br/><small>( "+
-				APILocator.getWorkflowAPI().findScheme(action.getSchemeId()).getName()
-				+" )</small>"%>
+		<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, action.getName()))%>
+		<%if(schemes!=null && schemes.size()>1  && contentlet.isNew()){ %>
```

Comment:
Hey @wezell I think the condition is wrong?, we can have contenlets already saved that are not yet in a step, I would leave the condition just as `schemes!=null && schemes.size()>1`
That makes sense to you?